integrations/paddleocr/tests/test_paddleocr_vl_document_converter.py

```diff
@@ -0,0 +1,784 @@
+# SPDX-FileCopyrightText: 2025-present deepset GmbH <info@deepset.ai>
+#
+# SPDX-License-Identifier: Apache-2.0
+import os
+from unittest.mock import MagicMock, patch
+
+import pypdfium2 as pdfium
+import pytest
+import requests
+from haystack import Document
+from haystack.dataclasses import ByteStream
+from haystack.utils import Secret
+from PIL import Image
+
+from haystack_integrations.components.converters.paddleocr import (
+    PaddleOCRVLDocumentConverter,
+)
+
+
+def download_test_file(url, dest_path, timeout=30):
```

Comment:
I would add the test files to this repo. See for example https://github.com/deepset-ai/haystack-core-integrations/tree/main/integrations/anthropic/tests/test_files

This would remove flakiness due to possible network issues.

Reply:
done